src/services/getWeatherData.js

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+
+export const getWeatherData = async (latitude, longitude, setWeatherInfo) => {
+  const wheatherAPI = `https://api.openweathermap.org/data/2.5/weather?lat=${latitude}&lon=${longitude}&appid=ff35f8965b5aa598cd3e3e536823bbad&units=metric`;
+
+  try {
+    const getData = await axios.get(wheatherAPI);
+    console.log(getData);

```

Comment:
You can remove comment if not necessary

Reply:
Done [b4475d4](https://github.com/Manpreet-hub/Browser-extension/pull/2/commits/b4475d420cfd72898055f4bf8b0952982862e7bb)